python/lib/import_dicom_study/import_log.py

```diff
@@ -1,3 +1,4 @@
+import getpass
```

Comment:
this is part of the default python install, right? No need for new packages? Just double checking